src/main/scala/Ch2.scala

```diff
@@ -0,0 +1,30 @@
+package swscala
+
+
+object Exercises1 {
+
+  val zeroToNine = (0 to 9)
+
+  // Problem 1
+  lazy val p1: Seq[(Int, Int)] = zeroToNine.flatMap(i => zeroToNine.map(j => (i, j)))
+    .filter{case (i: Int, j: Int) => i + 4 * j  > i * j}
+
+  // Problem 2
+  def p2(ss: Seq[String], bs: Seq[Boolean]): Seq[String] =
+    ss.zip(bs).filter{case (s: String, b: Boolean) => b}.map(_._1)
```

Comment:
Would it work to write `.filter(_._2)` instead of `.filter{case (s: String, b: Boolean) => b}`?

Reply:
Yup, I'll change it.